app/presenters/hyrax/object_resource_presenter.rb

```diff
@@ -0,0 +1,6 @@
+# Generated via
+#  `rails generate hyrax:work ObjectResource`
+module Hyrax
+  class ObjectResourcePresenter < Hyrax::WorkShowPresenter
```

Comment:
Similar to the question for the related test file. I'm wondering about the definition of this (empty) class.

Reply:
Good question!  It was automatically generated when run the task for new locales. I tried before that the tests would pass without these two classes: object_resource_presenter.rb and object_resource_indexer.rb. 
And I put them back to see if you would like to keep these templates or not?

Reply:
About Presenters in rails: http://nithinbekal.com/posts/rails-presenters/

Reply:
@hweng I think we've got both in horton:
https://github.com/ucsdlib/horton/blob/develop/app/presenters/object_show_presenter.rb
https://github.com/ucsdlib/horton/blob/develop/app/indexers/work_indexer.rb

Reply:
@lsitu Yes, Some naming convention concerns. As the model class is ObjectResource. Do we want to use the same convention for indexer and presenter, or keep our own name?

Reply:
@hweng I believe kind of code refactoring is needed if we want to change it. That could be related to https://github.com/ucsdlib/horton/issues/145 when moving forward.

Reply:
@lsitu Sounds good to me. It fine for me to remove that two classes in this pull request.